.github/workflows/builder.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           fetch-depth: 0
 
       - name: Set up Python ${{ env.DEFAULT_PYTHON }}
-        uses: actions/setup-python@v2.3.2
+        uses: actions/setup-python@v3
```

Comment:
This updated pinned just `v3`. Do we want to change that to `3.0.0`?

/CC: @frenck